.changeset/metal-impalas-allow.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/techdocs-common': patch
+---
+
+1. Techdocs publishers constructors now use parameter objects when being instantiated
+
+2. The `LocalPublish` publisher can now be created using `fromConfig`:
+
+```
```

Comment:
Probs want this to be 

```diff
- const publisher = new LocalPublish(config, logger, discovery);
+ const publisher = LocalPublish.fromConfig(config, logger, discovery);
```

Reply:
I was referencing the notation found in this _changelog_ which denotes a similar change.

https://github.com/backstage/backstage/blame/master/plugins/techdocs/CHANGELOG.md#L806

Is the preferred format to be a single `+` and `-`? Thanks.

Reply:
Think it's best to use single `+` and `-` 

And use 
```
```diff
```

Reply:
Gotcha - updated. Thanks!